CONTRIBUTING.md

```diff
@@ -153,6 +153,18 @@ get it merged. You can also post in the `#docs` [Slack
 channel](https://knative.slack.com/) to get input on your ideas or find areas to
 contribute before creating a PR.
 
+### Common Troubleshooting issues for PRs
+
+1. CLA fails even though you have signed a CLA. This can happen if someone accepts and commits suggestions due to the email address not matching the address on record for the CLA. 
```

Comment:
```suggestion
1. The CLA check fails even though you have signed the CLA. This may occur if you accept and commit suggestions in a pull request from another person's account, because the email address of that account doesn't match the address on record for the CLA. 
```

Reply:
IIUC this is generally because Github shows that commit as co-authored by the two parties (author and suggester) and uses their primary github email address as the commit email.

I'd love it if CLA bot didn't blow up 9/10 times on suggested edits.  It's honestly turned the feature from something I missing from Google's Critique to something I hardly think folks should use at all (cc @thisisnotapril , not sure what kinda juice you have with the team 😉 )